### ISSUE TITLE
Write a document explaining the features and about Bootstrap.

### ISSUE THREAD

**Issue by ADITIBAGADDEO22** [ADITIBAGADDEO22/Code-Non-Code#5]

**Comment by thechangamunda** [comment]
I would like to work on this assign this to me and do you want me to perform this in a readme.md format or just normal txt file format.

**Comment by ADITIBAGADDEO22** [comment]
@thechangamunda yes you can do it and add in the readme.md file as well as follow the sequence of the folders as per CONTRIBUTING.MD.
